webpack_loader/loader.py

```diff
@@ -16,25 +15,31 @@
 
 class WebpackLoader(object):
     _assets = {}
+    _cache_timestamp = 0
```

Comment:
Setting the timestamp in memory (as opposed to in redis) will mean every worker downloading the file every 30 seconds, which now that I think about it is exactly what we want for this architecture. Carry on.

Reply:
For v2 we could potentially implement a 2nd-level cache using the django cache framework (which in our case would put it in redis). Then we could rely on the cache's TTL functionality to only download the new file from s3 once every timeout cycle, instead of [number_of_web_workers] times. May need to be careful of dogpiling though.